crates/astria-sequencer/Cargo.toml

```diff
@@ -32,12 +32,14 @@ telemetry = { package = "astria-telemetry", path = "../astria-telemetry", featur
   "display",
 ] }
 
+anyhow = "1.0.89"
```

Comment:
There was a big effort to not have `anyhow` be a direct dependency of sequencer. Can we avoid reintroducing it here?

Reply:
I think we might be able to do that as part of #1436, but for now it's required since we're implementing [`cnidarium::StateRead`](https://rustdoc.penumbra.zone/main/cnidarium/trait.StateRead.html) on the new `Snapshot` type and that requires `anyhow` (`cnidarium` doesn't re-export it either).